frontend/packages/pipelines-plugin/src/components/taskruns/TaskRunDetailsStatus.tsx

```diff
@@ -7,13 +7,16 @@ import { taskRunFilterReducer } from '../../utils/pipeline-filter-reducer';
 import RunDetailsErrorLog from '../pipelineruns/logs/RunDetailsErrorLog';
 import { getTRLogSnippet } from './logs/taskRunLogSnippet';
 import { TaskRunKind } from '../../types';
+import WorkspaceResourceLinkList from '../shared/workspaces/WorkspaceResourceLinkList';
 
 export interface TaskRunDetailsStatusProps {
   taskRun: TaskRunKind;
 }
 
 const TaskRunDetailsStatus = ({ taskRun }) => {
   const { t } = useTranslation();
+  const ownerReference = taskRun.metadata.ownerReferences?.[0] || taskRun.metadata.name;
```

Comment:
`taskRun.metadata.ownerReferences?.[0]` This one is an object which is passed to `ownerResourceName` in `WorkspaceResourceLinkList` below. Shouldn't this be `taskRun.metadata.ownerReferences?.[0].name`  ?

Reply:
Thinking about this more, do we need this at all? TaskRun's ownerRef will be PipelineRun, but for `ownerResourceKind` is tied to taskrun.kind always, so it will never match :thinking: or am i missing something here?

ie: 
```
   <WorkspaceResourceLinkList
        workspaces={taskRun.spec.workspaces}
        namespace={taskRun.metadata.namespace}
        ownerResourceName={ownerReference} // <PipelineRun-name> (if taskrun is NOT standalone)
        ownerResourceKind={taskRun.kind} // TaskRun
      />
```

Reply:
yes, u r correct. I misplaced the oenerReference
```
      <WorkspaceResourceLinkList
        workspaces={taskRun.spec.workspaces}
        namespace={taskRun.metadata.namespace}
        ownerResourceName={ownerReference ? ownerReference.name : taskRun.metadata.name}
        ownerResourceKind={ownerReference ? ownerReference.kind : taskRun.kind}
      />
```